databases/week1/main.sql

```diff
@@ -0,0 +1,86 @@
+/*1*/
+SELECT
+SUM(1)
+FROM
+task;
+
+/*2*/
+SELECT
+SUM(1)
+FROM
+task
+WHERE
+due_date IS NULL;
+
+
+/*3*/
+SELECT  * 
+FROM
+ task 
+ JOIN status ON status.id= task.status_id
+ WHERE
+ status.name="Done";
```

Comment:
Works fine, could be simpler to notice that the 'Done' status has the id of 3.
So you could just query the task table where the task.status_id = 3.
That would also be faster if your table contained millions of tasks :)